examples/remarketing/add_logical_user_list.rb

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Creates a combination user list containing users that are present on any one
+# of the provided user lists.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def add_logical_user_list(customer_id, user_list_ids)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # Creates the UserListLogicalRuleInfo specifying that a user should be added
+  # to the new list if they are present in any of the provided lists.
+  user_list_logical_rule_info = client.resource.user_list_logical_rule_info do |info|
+    # Using ANY means that a user should be added to the combined list if they
+    # are present on any of the lists targeted in the LogicalUserListOperandInfo.
```

Comment:
If the class in Ruby is declared using PascalCase (LogicalUserListOperandInfo), it's fine.
If not, consider match this with the Ruby convention.

Reply:
changed to Ruby convention (snake case)